api/tests/test_building_api.py

```diff
@@ -31,6 +31,37 @@ def test_can_post_office_block(self, mock_verify_token):
         self.assertIn("name", response.data.keys())
         self.assertEqual(response.status_code, 201)
 
+    @patch('api.authentication.auth.verify_id_token')
+    def test_get_office_blocks_in_a_center_by_super_admin(self, mock_verify_token):
+        mock_verify_token.return_value = {'email': self.admin_user.email}
+        url = "/api/v1/andela-centres/{}/office_blocks/".format(self.centre.id)
+        token = "Token {}".format(self.token_user)
+        response = client.get(url, HTTP_AUTHORIZATION=token)
+        self.assertIn("name", response.data['results'][0].keys())
+        self.assertIn("id", response.data['results'][0].keys())
+        self.assertIn("location", response.data['results'][0].keys())
+        self.assertEqual(response.status_code, 200)
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_admin_can_get_office_block_in_their_center(self, mock_verify_token):
+        mock_verify_token.return_value = {'email': self.normal_admin.email}
+        url = "/api/v1/andela-centres/{}/office_blocks/".format(self.centre.id)
+        token = "Token {}".format(self.test_normaladmin)
+        response = client.get(url, HTTP_AUTHORIZATION=token)
+        response = client.get(url, HTTP_AUTHORIZATION=token)
+        self.assertIn("name", response.data['results'][0].keys())
+        self.assertIn("id", response.data['results'][0].keys())
+        self.assertIn("location", response.data['results'][0].keys())
+        self.assertEqual(response.status_code, 200)
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_cant_get_office_block_in_center_by_non_superuser(self, mock_verify_token):
+        mock_verify_token.return_value = {'email': self.other_user.email}
+        url = "/api/v1/andela-centres/{}/office_blocks/".format(self.other_centre.id)
```

Comment:
In this and the rest of the tests you could point to the generated url using `rest_framework.reverse.reverse` while referencing the name of the url. The name would be `andela-centres-office-blocks`: reverse('andela-centres-office-blocks'). You could define the reverse url as a variable in the __init__ file of the tests just like the others.

Reply:
done